core/src/sigverifier/bls_sigverifier.rs

```diff
@@ -90,17 +51,16 @@ impl SigVerifier for BLSSigVerifier {
         packet_batches: Vec<PacketBatch>,
     ) -> Result<(), SigVerifyServiceError<Self::SendType>> {
         // TODO(wen): just a placeholder without any batching.
-        let mut stats = BLSSigVerifierStats::new();
-        let mut verified_votes: HashMap<Pubkey, Vec<Slot>> = HashMap::new();
+        let mut verified_votes = HashMap::new();
 
         for packet in packet_batches.iter().flatten() {
-            stats.received += 1;
+            self.stats.received += 1;
```

Comment:
We are allocating a new stats every time because we want to do parallel processing later.

Reply:
Thanks for the feeback @wen-coding.  I've made some improvements based on your feedback.  PTAL.